### ISSUE TITLE
Ever saw this ?

### ISSUE THREAD

**Issue by Taverne** [dalaranwow/dalaran-wow#4867]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Cannot connect
**Expected behaviour**: The game should give me the realm list so I can create my 1st character

**Steps to reproduce the problem**:

1. Well, I don't know, check the screenshots
2. 
3. 

**Include proofs for this behaviour**

https://ibb.co/iMc3Bk
https://ibb.co/g6aC45

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Taverne** [comment]
Ok my bad, I was doing this the wrong way. Sorry for that

**Comment by Crispybuilt** [comment]
Log with your username and not e-mail.
Close up the issue

**Comment by TeridanDW** [comment]
> Log with your username and not e-mail.
